config/initializers/s3_client.rb

```diff
@@ -10,13 +10,13 @@ def lookup_mock_data(key)
   end
 end
 
-if Rails.env.production?
-  S3_BUCKET = Aws::S3::Client.new(region: 'eu-west-1')
-else
-  S3_BUCKET = Aws::S3::Client.new(region: 'eu-west-1', stub_responses: true)
+if Rails.env.test?
+  S3_BUCKET = Aws::S3::Client.new(region: 'eu-west-2', stub_responses: true)
```

Comment:
How come we're changing this from eu-west-1 to eu-west-2?

Reply:
the metrics are done in London which is `eu-west-2`, that's where the bucket lives.